_shared/project/tox.ini

```diff
@@ -69,12 +70,13 @@ deps =
     format,checkformatting: black
     format,checkformatting: isort
     lint: toml
-    lint: pylint
+    lint: pylint>=3.0.0
     lint: pydocstyle
     lint: pycodestyle
     lint,tests: pytest-mock
-    lint,tests,coverage: coverage[toml]
     lint,tests,functests: pytest
+    tests: pytest-coverage
+    coverage: coverage[toml]
```

Comment:
The `coverage` environment still depends on the normal `coverage` command line tool, not `pytest-coverage` (that's only needed for running the tests)